diff -aurN gnome-media-2.14.2.orig/cddb-slave2/cddb-slave.desktop.in.in gnome-media-2.14.2/cddb-slave2/cddb-slave.desktop.in.in
--- gnome-media-2.14.2.orig/cddb-slave2/cddb-slave.desktop.in.in	2006-01-21 21:22:50.000000000 +0100
+++ gnome-media-2.14.2/cddb-slave2/cddb-slave.desktop.in.in	2006-05-31 20:55:54.177216648 +0200
@@ -7,7 +7,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;Settings;AdvancedSettings;
+Categories=GTK;GNOME;Settings;AdvancedSettings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
diff -aurN gnome-media-2.14.2.orig/gnome-cd/gnome-cd.desktop.in.in gnome-media-2.14.2/gnome-cd/gnome-cd.desktop.in.in
--- gnome-media-2.14.2.orig/gnome-cd/gnome-cd.desktop.in.in	2006-05-29 17:29:05.000000000 +0200
+++ gnome-media-2.14.2/gnome-cd/gnome-cd.desktop.in.in	2006-05-31 20:44:30.118209424 +0200
@@ -8,7 +8,7 @@
 Terminal=false
 Type=Application
 X-GNOME-DocPath=gnome-cd/gnome-cd.xml
-Categories=GNOME;GTK;AudioVideo;Audio;Player;X-Ximian-Main;X-Red-Hat-Base;
+Categories=GTK;GNOME;AudioVideo;Audio;Player;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
 X-GNOME-Bugzilla-Component=Gnome-CD
diff -aurN gnome-media-2.14.2.orig/grecord/gnome-sound-recorder.desktop.in.in gnome-media-2.14.2/grecord/gnome-sound-recorder.desktop.in.in
--- gnome-media-2.14.2.orig/grecord/gnome-sound-recorder.desktop.in.in	2006-05-29 17:29:05.000000000 +0200
+++ gnome-media-2.14.2/grecord/gnome-sound-recorder.desktop.in.in	2006-05-31 20:44:45.081934592 +0200
@@ -8,7 +8,7 @@
 Terminal=false
 Type=Application
 X-GNOME-DocPath=grecord/grecord.xml
-Categories=GNOME;GTK;AudioVideo;Recorder;
+Categories=GTK;GNOME;AudioVideo;Audio;Recorder;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
 X-GNOME-Bugzilla-Component=Gnome-Sound-Recorder
diff -aurN gnome-media-2.14.2.orig/gst-mixer/gnome-volume-control.desktop.in.in gnome-media-2.14.2/gst-mixer/gnome-volume-control.desktop.in.in
--- gnome-media-2.14.2.orig/gst-mixer/gnome-volume-control.desktop.in.in	2006-01-21 21:22:51.000000000 +0100
+++ gnome-media-2.14.2/gst-mixer/gnome-volume-control.desktop.in.in	2006-05-31 20:45:03.269169712 +0200
@@ -8,7 +8,7 @@
 X-Info=Gnome Mixer
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;AudioVideo;X-Ximian-Main;X-Red-Hat-Base;
+Categories=GTK;GNOME;AudioVideo;Audio;Mixer;
 X-GNOME-DocPath=gst-mixer/gnome-volume-control.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
diff -aurN gnome-media-2.14.2.orig/gstreamer-properties/gstreamer-properties.desktop.in.in gnome-media-2.14.2/gstreamer-properties/gstreamer-properties.desktop.in.in
--- gnome-media-2.14.2.orig/gstreamer-properties/gstreamer-properties.desktop.in.in	2006-01-21 21:22:52.000000000 +0100
+++ gnome-media-2.14.2/gstreamer-properties/gstreamer-properties.desktop.in.in	2006-05-31 20:56:05.925430648 +0200
@@ -7,7 +7,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;Settings;AdvancedSettings;
+Categories=GTK;GNOME;Settings;AdvancedSettings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
diff -aurN gnome-media-2.14.2.orig/vu-meter/reclevel.desktop.in.in gnome-media-2.14.2/vu-meter/reclevel.desktop.in.in
--- gnome-media-2.14.2.orig/vu-meter/reclevel.desktop.in.in	2006-01-21 21:22:52.000000000 +0100
+++ gnome-media-2.14.2/vu-meter/reclevel.desktop.in.in	2006-05-31 20:46:09.034171912 +0200
@@ -6,4 +6,4 @@
 Icon=gnome-reclevel.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;AudioVideo;
+Categories=GTK;GNOME;AudioVideo;Audio;
diff -aurN gnome-media-2.14.2.orig/vu-meter/vumeter.desktop.in.in gnome-media-2.14.2/vu-meter/vumeter.desktop.in.in
--- gnome-media-2.14.2.orig/vu-meter/vumeter.desktop.in.in	2006-01-21 21:22:52.000000000 +0100
+++ gnome-media-2.14.2/vu-meter/vumeter.desktop.in.in	2006-05-31 20:46:18.512730952 +0200
@@ -7,7 +7,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;AudioVideo;
+Categories=GTK;GNOME;AudioVideo;Audio;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
 X-GNOME-Bugzilla-Component=VU-meter
