diff -urN gnome-media-2.0.0.orig/cddb-slave2/Makefile.am gnome-media-2.0.0/cddb-slave2/Makefile.am
--- gnome-media-2.0.0.orig/cddb-slave2/Makefile.am	Fri Aug 16 23:02:03 2002
+++ gnome-media-2.0.0/cddb-slave2/Makefile.am	Fri Aug 16 23:51:52 2002
@@ -30,7 +30,6 @@
 idl_DATA = $(IDLS)
 
 CDDBSlave2_SOURCES =				\
-	$(IDL_GENERATED)			\
 	cddb-slave.c				\
 	cddb-slave.h				\
 	cddb-parser.c				\
@@ -51,12 +50,12 @@
 	tcp.h
 
 CDDBSlave2_LDADD =				\
-	$(CDDBSLAVE_LIBS)				\
-	$(INTLLIBS)
+	$(CDDBSLAVE_LIBS)			\
+	$(INTLLIBS)				\
+	libcddb-slave2.la
 
 cddb_track_editor_SOURCES = 			\
-	$(IDL_GENERATED)			\
-	cddb-disclosure.c	\
+	cddb-disclosure.c			\
 	cddb-disclosure.h			\
 	cddb-track-editor.c
 
@@ -105,6 +104,5 @@
 EXTRA_DIST = $(oaf_in_files) $(Desktop_in_files) $(oaf_DATA) $(IDLS) $(schema_DATA)
 
 install-schemas:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 
 install-data-local: install-schemas
diff -urN gnome-media-2.0.0.orig/gmix/Makefile.am gnome-media-2.0.0/gmix/Makefile.am
--- gnome-media-2.0.0.orig/gmix/Makefile.am	Fri Aug 16 23:02:03 2002
+++ gnome-media-2.0.0/gmix/Makefile.am	Fri Aug 16 23:52:01 2002
@@ -36,7 +36,6 @@
 schema_DATA=gnome-volume-control.schemas
 
 install-schemas:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 
 install-data-local: install-schemas
 endif
diff -urN gnome-media-2.0.0.orig/gnome-cd/Makefile.am gnome-media-2.0.0/gnome-cd/Makefile.am
--- gnome-media-2.0.0.orig/gnome-cd/Makefile.am	Fri Aug 16 23:02:03 2002
+++ gnome-media-2.0.0/gnome-cd/Makefile.am	Fri Aug 16 23:52:23 2002
@@ -70,6 +70,5 @@
 schema_DATA=gnome-cd.schemas
 
 install-schemas:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(top_builddir)/gnome-cd/gnome-cd.schemas 
 
 install-data-local: install-schemas
diff -urN gnome-media-2.0.0.orig/grecord/Makefile.am gnome-media-2.0.0/grecord/Makefile.am
--- gnome-media-2.0.0.orig/grecord/Makefile.am	Fri Aug 16 23:02:03 2002
+++ gnome-media-2.0.0/grecord/Makefile.am	Fri Aug 16 23:52:32 2002
@@ -26,7 +26,6 @@
 schema_DATA=gnome-sound-recorder.schemas
 
 install-schemas:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(schema_DATA)
 
 install-data-local: install-schemas
 
