diff -urN aaa/gnome-media-2.4.1.1/gmix/gnome-volume-control.desktop.in gnome-media-2.4.1.1/gmix/gnome-volume-control.desktop.in
--- aaa/gnome-media-2.4.1.1/gmix/gnome-volume-control.desktop.in	2003-10-15 19:34:45.000000000 +0200
+++ gnome-media-2.4.1.1/gmix/gnome-volume-control.desktop.in	2003-12-12 18:35:06.000000000 +0100
@@ -4,8 +4,9 @@
 Exec=gnome-volume-control
 Icon=gnome-mixer.png
 StartupNotify=true
-Info=Gnome Mixer
-Terminal=0
+X-Info=Gnome Mixer
+Terminal=false
 Type=Application
 Categories=Application;AudioVideo;
 X-GNOME-DocPath=gmix/gmix.xml
+Encoding=UTF-8
diff -urN aaa/gnome-media-2.4.1.1/gmix/gnome-volume-control.desktop.in.in gnome-media-2.4.1.1/gmix/gnome-volume-control.desktop.in.in
--- aaa/gnome-media-2.4.1.1/gmix/gnome-volume-control.desktop.in.in	2002-12-11 18:32:15.000000000 +0100
+++ gnome-media-2.4.1.1/gmix/gnome-volume-control.desktop.in.in	2003-12-12 18:34:49.000000000 +0100
@@ -4,8 +4,9 @@
 Exec=@DEFAULT_MIXER@
 Icon=gnome-mixer.png
 StartupNotify=true
-Info=Gnome Mixer
-Terminal=0
+X-Info=Gnome Mixer
+Terminal=false
 Type=Application
 Categories=Application;AudioVideo;
 X-GNOME-DocPath=gmix/gmix.xml
+Encoding=UTF-8
