diff -Nru gnome-media-1.2.0.orig/tcd/Makefile.am gnome-media-1.2.0/tcd/Makefile.am
--- gnome-media-1.2.0.orig/tcd/Makefile.am	Fri May 26 00:11:10 2000
+++ gnome-media-1.2.0/tcd/Makefile.am	Fri May 26 00:11:23 2000
@@ -1,4 +1,4 @@
-Audiodir = $(datadir)/gnome/apps/Multimedia
+Audiodir = $(datadir)/applnk/Multimedia
 Pixmapdir = $(datadir)/pixmaps
 
 EXTRA_DIST = README TODO NEWS gtcd.desktop gtcd.png
diff -Nru gnome-media-1.2.0.orig/gmix/Makefile.am gnome-media-1.2.0/gmix/Makefile.am
--- gnome-media-1.2.0.orig/gmix/Makefile.am	Fri May 26 02:07:20 2000
+++ gnome-media-1.2.0/gmix/Makefile.am	Fri May 26 02:10:55 2000
@@ -1,4 +1,4 @@
-Audiodir = $(datadir)/gnome/apps/Multimedia
+Audiodir = $(datadir)/applnk/Multimedia
 pixmapdir = $(datadir)/pixmaps
 
 EXTRA_DIST = README TODO NEWS gmix.desktop gnome-mixer.png \
diff -Nru gnome-media-1.2.0.orig/grecord/Makefile.am gnome-media-1.2.0/grecord/Makefile.am
--- gnome-media-1.2.0.orig/grecord/Makefile.am	Fri May 26 02:07:20 2000
+++ gnome-media-1.2.0/grecord/Makefile.am	Fri May 26 02:10:48 2000
@@ -4,7 +4,7 @@
 	Soundrecorder.desktop README COPYING AUTHORS INSTALL \
 	pixmaps/gnome_mixer.xpm grecord.keys.in grecord.mime
 
-Applicationsdir = $(datadir)/gnome/apps/Multimedia
+Applicationsdir = $(datadir)/applnk/Multimedia
 Applications_DATA = Soundrecorder.desktop
 
 Mimedir = $(datadir)/mime-info
@@ -12,4 +12,4 @@
 
 grecord.keys: grecord.keys.in Makefile
 	sed -e 's^\@icondir\@^$(appicondir)^g' < $(srcdir)/grecord.keys.in > grecord.keys.tmp \
-	&& mv grecord.keys.tmp grecord.keys 
\ No newline at end of file
+	&& mv grecord.keys.tmp grecord.keys 
diff -Nru gnome-media-1.2.0.orig/vu-meter/Makefile.am gnome-media-1.2.0/vu-meter/Makefile.am
--- gnome-media-1.2.0.orig/vu-meter/Makefile.am	Fri May 26 02:07:20 2000
+++ gnome-media-1.2.0/vu-meter/Makefile.am	Fri May 26 02:10:58 2000
@@ -1,4 +1,4 @@
-Audiodir = $(datadir)/gnome/apps/Multimedia
+Audiodir = $(datadir)/applnk/Multimedia
 
 pixmapdir = $(datadir)/pixmaps
 
