diff -urN aa/gnome-media-2.5.2/profiles/gnome-audio-profiles.schemas.in.in gnome-media-2.5.2/profiles/gnome-audio-profiles.schemas.in.in
--- aa/gnome-media-2.5.2/profiles/gnome-audio-profiles.schemas.in.in	2003-12-17 12:05:00.000000000 +0100
+++ gnome-media-2.5.2/profiles/gnome-audio-profiles.schemas.in.in	2004-01-31 14:16:05.877431648 +0100
@@ -79,7 +79,7 @@
       <key>/schemas/system/gstreamer/audio/profiles/active</key>
       <owner>gnome-media</owner>
       <type>bool</type>
-      <default></default>
+      <default>true</default>
       <locale name="C">
         <short>Whether or not this profile is to be used</short>
         <long>
