diff -urN gnome-media-2.8.0.orig/gnome-cd/gnome-cd.desktop.in gnome-media-2.8.0/gnome-cd/gnome-cd.desktop.in
--- gnome-media-2.8.0.orig/gnome-cd/gnome-cd.desktop.in	2004-11-17 14:14:29.240095904 +0100
+++ gnome-media-2.8.0/gnome-cd/gnome-cd.desktop.in	2004-11-17 14:25:24.753442728 +0100
@@ -8,7 +8,7 @@
 Terminal=false
 Type=Application
 X-GNOME-DocPath=gnome-cd/gnome-cd.xml
-Categories=GNOME;Application;AudioVideo;X-Ximian-Main;X-Red-Hat-Base;
+Categories=GTK;GNOME;AudioVideo;Audio;Player;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
 X-GNOME-Bugzilla-Component=Gnome-CD
diff -urN gnome-media-2.8.0.orig/grecord/gnome-sound-recorder.desktop.in gnome-media-2.8.0/grecord/gnome-sound-recorder.desktop.in
--- gnome-media-2.8.0.orig/grecord/gnome-sound-recorder.desktop.in	2004-11-17 14:14:29.037126760 +0100
+++ gnome-media-2.8.0/grecord/gnome-sound-recorder.desktop.in	2004-11-17 14:17:57.999359672 +0100
@@ -8,7 +8,7 @@
 Terminal=false
 Type=Application
 X-GNOME-DocPath=grecord/grecord.xml
-Categories=GNOME;Application;AudioVideo;
+Categories=GTK;GNOME;AudioVideo;Audio;Recorder;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
 X-GNOME-Bugzilla-Component=Gnome-Sound-Recorder
diff -urN gnome-media-2.8.0.orig/gst-mixer/gnome-volume-control.desktop.in gnome-media-2.8.0/gst-mixer/gnome-volume-control.desktop.in
--- gnome-media-2.8.0.orig/gst-mixer/gnome-volume-control.desktop.in	2004-11-17 14:14:29.225098184 +0100
+++ gnome-media-2.8.0/gst-mixer/gnome-volume-control.desktop.in	2004-11-17 14:20:42.871295328 +0100
@@ -8,7 +8,7 @@
 X-Info=Gnome Mixer
 Terminal=false
 Type=Application
-Categories=Application;AudioVideo;X-Ximian-Main;X-Red-Hat-Base;
+Categories=GTK;GNOME;AudioVideo;Audio;Mixer;
 X-GNOME-DocPath=gst-mixer/gnome-volume-control.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
diff -urN gnome-media-2.8.0.orig/vu-meter/reclevel.desktop.in gnome-media-2.8.0/vu-meter/reclevel.desktop.in
--- gnome-media-2.8.0.orig/vu-meter/reclevel.desktop.in	2004-11-17 14:14:29.083119768 +0100
+++ gnome-media-2.8.0/vu-meter/reclevel.desktop.in	2004-11-17 14:23:34.778161512 +0100
@@ -6,4 +6,4 @@
 Icon=gnome-reclevel.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;AudioVideo;
+Categories=GTK;GNOME;AudioVideo;Audio;
diff -urN gnome-media-2.8.0.orig/vu-meter/vumeter.desktop.in gnome-media-2.8.0/vu-meter/vumeter.desktop.in
--- gnome-media-2.8.0.orig/vu-meter/vumeter.desktop.in	2004-11-17 14:14:29.082119920 +0100
+++ gnome-media-2.8.0/vu-meter/vumeter.desktop.in	2004-11-17 14:21:39.081750040 +0100
@@ -7,7 +7,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;Application;AudioVideo;
+Categories=GTK;GNOME;AudioVideo;Audio;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
 X-GNOME-Bugzilla-Component=VU-meter
