diff -urN gnome-media-2.17.91/configure.in gnome-media-2.17.91.new/configure.in
--- gnome-media-2.17.91/configure.in	2007-02-12 15:06:29.000000000 +0100
+++ gnome-media-2.17.91.new/configure.in	2007-03-05 16:17:04.000000000 +0100
@@ -178,51 +178,8 @@
 
 if test "x$HAVE_GST" = "xyes" && test "x$enable_grecord" == "xyes"; then
   PROGRAMS_GRECORD="grecord"
-
-  gst010_toolsdir=`$PKG_CONFIG --variable=toolsdir gstreamer-0.10`
-  gst010_inspect="$gst010_toolsdir/gst-inspect-0.10"
-
-  dnl Check for elements from gst-plugins-base
-  for base_element in playbin audioconvert audioresample
-  do
-    AC_MSG_CHECKING([for GStreamer 0.10 $base_element plugin])
-    if $gst010_inspect $base_element >/dev/null 2>/dev/null; then
-      AC_MSG_RESULT([yes])
-    else
-      AC_MSG_RESULT([no])
-      AC_MSG_WARN([
-                    Cannot find required GStreamer-0.10 plugin '$base_element'.
-                    It should be part of gst-plugins-base. Please install it.
-                    You need it installed for the sound recorder to work.
-      ])
-      PROGRAMS_GRECORD=""
-      PROGRAMS_GRECORD_ERROR="Missing gst-plugins-base '$base_element' element\n"
-    fi
-  done
-
-
-  dnl Check for elements from gst-plugins-good
-  for good_element in gconfaudiosink gconfaudiosrc
-  do
-    AC_MSG_CHECKING([for GStreamer 0.10 $good_element plugin])
-    if $gst010_inspect $good_element >/dev/null 2>/dev/null; then
-      AC_MSG_RESULT([yes])
-    else
-      AC_MSG_RESULT([no])
-      AC_MSG_WARN([
-                   Cannot find required GStreamer-0.10 plugin '$good_element'.
-                   It should be part of gst-plugins-good. Please install it.
-                   You need it installed for the sound recorder to work.
-      ])
-      PROGRAMS_GRECORD=""
-      PROGRAMS_GRECORD_ERROR="Missing gst-plugins-good '$good_element' element\n"
-    fi
-  done
 else
-  if test "x$HAVE_GST" = "xno"; then		
-    AC_MSG_WARN([You need GStreamer installed for the mixer and recorder to work])
-    sleep 5 2>/dev/null
-  fi	
+  PROGRAMS_GRECORD=""
 fi
 
 AC_SUBST(PROGRAMS_GRECORD)
