diff -Nru gnome-media-2.5.2.orig/configure.in gnome-media-2.5.2/configure.in
--- gnome-media-2.5.2.orig/configure.in	2003-12-31 04:08:24.000000000 +0100
+++ gnome-media-2.5.2/configure.in	2003-12-31 11:39:30.805607064 +0100
@@ -88,7 +88,7 @@
 if test "x$HAVE_GST" = "xno"; then
   AC_MSG_WARN(you need GStreamer installed for the mixer and recorder to work)
 else
-  PROGRAMS_GSTRECORD="grecord"
+  PROGRAMS_GRECORD="grecord"
 fi
 AC_SUBST(PROGRAMS_GRECORD)
 
