diff -ur gnome-media-1.2.3.orig/configure.in gnome-media-1.2.3/configure.in
--- gnome-media-1.2.3.orig/configure.in	Sat Jan 10 11:49:04 2004
+++ gnome-media-1.2.3/configure.in	Sat Jan 10 11:50:55 2004
@@ -69,7 +69,7 @@
     fi
 fi
 
-ALL_LINGUAS="az ca cs da de el en_GB es et fi fr ga gl hu it ja ko lt nl no pl pt pt_BR ro ru sk sl sv tr uk zh_TW.Big5 zh_CN.GB2312"
+ALL_LINGUAS="az ca cs da de el en_GB es et fi fr ga gl hu it ja ko lt nl no pl pt pt_BR ro ru sk sl sv tr uk zh_TW zh_CN"
 AM_GNU_GETTEXT
 
 AC_SUBST(CFLAGS)
