diff -Nru gnome-media-2.2.1.orig/configure.in gnome-media-2.2.1/configure.in
--- gnome-media-2.2.1.orig/configure.in	Wed Feb  5 00:48:09 2003
+++ gnome-media-2.2.1/configure.in	Wed Feb  5 10:46:09 2003
@@ -313,7 +313,7 @@
 		  PROGRAMS_GSTPROPS="gstreamer-properties",
 		  PROGRAMS_GSTPROPS="")
 
-GSTPROPS_LIBS="$GSTPROPS_LIBS $x_libs -lgstgconf-0.5"
+GSTPROPS_LIBS="$GSTPROPS_LIBS $x_libs -lgstgconf-0.6"
 AC_SUBST(GSTPROPS_LIBS)
 AC_SUBST(GSTPROPS_CFLAGS)
 
