diff -urN gnome-media-2.18.0/grecord/gnome-sound-recorder.desktop.in.in gnome-media-2.18.0.new/grecord/gnome-sound-recorder.desktop.in.in
--- gnome-media-2.18.0/grecord/gnome-sound-recorder.desktop.in.in	2007-01-06 19:16:58.000000000 +0100
+++ gnome-media-2.18.0.new/grecord/gnome-sound-recorder.desktop.in.in	2007-03-13 12:43:27.000000000 +0100
@@ -8,7 +8,7 @@
 Terminal=false
 Type=Application
 X-GNOME-DocPath=grecord/grecord.xml
-Categories=GNOME;GTK;AudioVideo;Recorder;
+Categories=GNOME;GTK;AudioVideo;Audio;Recorder;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-media
 X-GNOME-Bugzilla-Component=Gnome-Sound-Recorder
