--- gnome-media-2.18.0/gstreamer-properties/pipeline-constants.c~	2007-02-12 14:33:28.000000000 +0100
+++ gnome-media-2.18.0/gstreamer-properties/pipeline-constants.c	2007-09-03 09:02:18.102391323 +0200
@@ -99,6 +99,8 @@
   {PIPE_TYPE_VIDEOSINK, 0, "Colour Ascii Art", "cacasink", NULL, FALSE,
       TEST_PIPE_SUPPLIED, videosink_test_pipe, FALSE},
 #endif
+  {PIPE_TYPE_VIDEOSINK, 0, N_("OpenGL"), "glimagesink", NULL, FALSE,
+      TEST_PIPE_SUPPLIED, videosink_test_pipe, FALSE},
   {PIPE_TYPE_VIDEOSINK, 0, N_("SDL - Simple DirectMedia Layer"), "sdlvideosink",
       NULL, FALSE, TEST_PIPE_SUPPLIED, videosink_test_pipe, FALSE},
   {PIPE_TYPE_VIDEOSINK, 0, N_("X Window System (No Xv)"),
