--- gnome-media-1.2.0/tcd/Makefile.am.gtcdcorba	Sun May 21 16:13:04 2000
+++ gnome-media-1.2.0/tcd/Makefile.am	Thu Jun 15 17:57:08 2000
@@ -1,7 +1,7 @@
 Audiodir = $(datadir)/gnome/apps/Multimedia
 Pixmapdir = $(datadir)/pixmaps
 
-EXTRA_DIST = README TODO NEWS gtcd.desktop gtcd.png
+EXTRA_DIST = README TODO NEWS gtcd.desktop gtcd.png gtcd.goad gtcd.idl
 
 # Should make sure that tcd will work with all flavors of curses.
 # Make sure it can support curses w/o color and old versions.
@@ -9,10 +9,17 @@
 CURSES_TCD=tcd
 endif
 
-bin_PROGRAMS = gtcd $(CURSES_TCD)
+bin_PROGRAMS = gtcd $(CURSES_TCD) corba-gtcd
 
 SUBDIRS = icons
 
+
+GTCD_GENERATED = 			\
+		gtcd.h   		\
+		gtcd-stubs.c	\
+		gtcd-skels.c 	\
+		gtcd-common.c
+
 gtcd_SOURCES = 		\
 	gtcd.c 		\
 	cddb.c 		\
@@ -34,7 +41,16 @@
 	gtcd_public.h	\
 	cddb_status.c	\
 	cddb_props.c	\
-	cddb_props.h
+	cddb_props.h	\
+	gtcd-skels.c     \
+	gtcd-common.c    \
+	gtcd.h
+
+corba_gtcd_SOURCES = \
+	corba-gtcd.c \
+	gtcd.h		\
+	gtcd-stubs.c	\
+	gtcd-common.c
 
 tcd_SOURCES = 		\
 	tcd.c 		\
@@ -47,16 +63,32 @@
 	tracked.c
 
 gtcd_LDADD = 	$(GNOME_LIBDIR) \
-		$(GNOMEUI_LIBS) \
+		$(GTCD_GNOME_LIBS) \
 		$(INTLLIBS)
 
+corba_gtcd_LDADD = 	$(GNOME_LIBDIR) \
+			$(GTCD_GNOME_LIBS) \
+			$(INTLLIBS)
+
 tcd_LDADD = $(GNOME_LIBDIR) -lglib -lm $(CURSES_LIBS) $(INTLLIBS)
 
 CLEANFILES = *~
 
-INCLUDES = -I$(includedir) $(GNOME_INCLUDEDIR) $(CURSES_INCLUDEDIR) \
+INCLUDES = -I$(includedir) $(GTCD_GNOME_CFLAGS) $(CURSES_INCLUDEDIR) \
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" $(USE_CD_CHANGER)	\
 	-DGNOME_ICONDIR=\""$(Pixmapdir)"\"
 
 Audio_DATA = gtcd.desktop
-Pixmap_DATA = gtcd.png
\ No newline at end of file
+Pixmap_DATA = gtcd.png
+
+idldir = $(datadir)/idl
+idl_DATA = gtcd.idl
+
+goaddir = $(sysconfdir)/CORBA/servers
+goad_DATA = gtcd.goad
+
+BUILT_SOURCES = $(GTCD_GENERATED)
+
+$(GTCD_GENERATED): gtcd.idl
+	orbit-idl $(GNOME_IDLFLAGS) gtcd.idl
+
--- gnome-media-1.2.0/tcd/gtcd.c.gtcdcorba	Sun May 21 16:13:04 2000
+++ gnome-media-1.2.0/tcd/gtcd.c	Thu Jun 15 17:53:01 2000
@@ -42,6 +42,8 @@
 #include <gnome.h>
 #include <gdk/gdkkeysyms.h>
 
+#include <libgnorba/gnorba.h>
+
 #include "linux-cdrom.h"
 
 #include "gtracked.h"
@@ -50,6 +52,7 @@
 #include "callbacks.h"
 #include "gtcd_public.h"
 #include "keybindings.h"
+#include "gtcd.h"
 
 /* time display types */
 enum { TIME_FIRST=-1, TRACK_E, TRACK_R, DISC_E, DISC_R, TIME_LAST };
@@ -997,13 +1000,95 @@
 	default:
 		break;
 	}
-}	
+}
 
 void setup_keys()
 {
-	accel = gtk_accel_group_get_default();
+        accel = gtk_accel_group_get_default();
 }    
 
+/*
+ * CORBA glue
+ */
+
+/*** App-specific servant structures ***/
+typedef struct {
+   POA_GNOME_GTcd servant;
+   PortableServer_POA poa;
+
+} impl_POA_GNOME_GTcd;
+
+/*** Implementation stub prototypes ***/
+static void impl_GNOME_GTcd__destroy(impl_POA_GNOME_GTcd * servant,
+				     CORBA_Environment * ev);
+static void
+impl_GNOME_GTcd_open_new_cd(impl_POA_GNOME_GTcd * servant,
+                            CORBA_Environment * ev);
+
+/*** epv structures ***/
+static PortableServer_ServantBase__epv impl_GNOME_GTcd_base_epv =
+{
+   NULL,			/* _private data */
+   NULL,			/* finalize routine */
+   NULL,			/* default_POA routine */
+};
+static POA_GNOME_GTcd__epv impl_GNOME_GTcd_epv =
+{
+   NULL,			/* _private */
+   (gpointer) & impl_GNOME_GTcd_open_new_cd,
+
+};
+
+/*** vepv structures ***/
+static POA_GNOME_GTcd__vepv impl_GNOME_GTcd_vepv =
+{
+   &impl_GNOME_GTcd_base_epv,
+   &impl_GNOME_GTcd_epv,
+};
+
+/*** Stub implementations ***/
+static GNOME_GTcd 
+impl_GNOME_GTcd__create(PortableServer_POA poa, CORBA_Environment * ev)
+{
+   GNOME_GTcd retval;
+   impl_POA_GNOME_GTcd *newservant;
+   PortableServer_ObjectId *objid;
+
+   newservant = g_new0(impl_POA_GNOME_GTcd, 1);
+   newservant->servant.vepv = &impl_GNOME_GTcd_vepv;
+   newservant->poa = poa;
+   POA_GNOME_GTcd__init((PortableServer_Servant) newservant, ev);
+   objid = PortableServer_POA_activate_object(poa, newservant, ev);
+   CORBA_free(objid);
+   retval = PortableServer_POA_servant_to_reference(poa, newservant, ev);
+
+   return retval;
+}
+
+static void
+impl_GNOME_GTcd__destroy(impl_POA_GNOME_GTcd * servant, CORBA_Environment * ev)
+{
+   PortableServer_ObjectId *objid;
+
+   objid = PortableServer_POA_servant_to_id(servant->poa, servant, ev);
+   PortableServer_POA_deactivate_object(servant->poa, objid, ev);
+   CORBA_free(objid);
+
+   POA_GNOME_GTcd__fini((PortableServer_Servant) servant, ev);
+   g_free(servant);
+}
+
+static void
+impl_GNOME_GTcd_open_new_cd(impl_POA_GNOME_GTcd * servant,
+			    CORBA_Environment * ev)
+{
+	tcd_playtracks(&cd, cd.first_t, cd.last_t, prefs->only_use_trkind);
+}
+
+/*
+ * Main
+ */
+
 static char *CD_device = NULL;
 poptContext ctx;
 
@@ -1021,12 +1106,16 @@
 {
 	GtkWidget *table;
 	GdkColor black;
-
+        CORBA_ORB orb;
+        CORBA_Environment ev;
+        PortableServer_POA poa;
+        
 	bindtextdomain(PACKAGE, GNOMELOCALEDIR);
 	textdomain(PACKAGE);
 
-	gnome_init_with_popt_table("gtcd", VERSION, argc, argv, gtcd_popt_options, 0, &ctx);
-
+        CORBA_exception_init(&ev);
+	orb = gnome_CORBA_init_with_popt_table("gtcd", VERSION, &argc, argv, gtcd_popt_options, 0, &ctx, GNORBA_INIT_SERVER_FUNC, &ev);
+        
 	cd.play_method = NORMAL;
 
 	gnome_window_icon_set_default_from_file (GNOME_ICONDIR"/gtcd.png");
@@ -1072,6 +1161,16 @@
     
 	signal(SIGUSR2, reload_info);
     
+         poa = (PortableServer_POA)CORBA_ORB_resolve_initial_references(orb, "RootPOA", &ev);
+         PortableServer_POAManager_activate(PortableServer_POA__get_the_POAManager(poa, &ev), &ev);
+         goad_server_register(CORBA_OBJECT_NIL,
+                              impl_GNOME_GTcd__create(poa, &ev),
+                              "GOAD:gtcd:19990918", "object", &ev);
+ 
+         
 	gtk_main();
 	return 0;
 }
+
+
+
--- gnome-media-1.2.0/tcd/callbacks.c.gtcdcorba	Thu Jan 13 23:27:52 2000
+++ gnome-media-1.2.0/tcd/callbacks.c	Thu Jun 15 17:53:01 2000
@@ -2,6 +2,7 @@
 
 #include <config.h>
 #include <gnome.h>
+#include <libgnorba/gnorba.h>
 #include <sys/types.h>
 
 #include "linux-cdrom.h"
@@ -117,7 +118,13 @@
 	gdk_gc_destroy(gc);
 	exit_action();
 	tcd_close_disc(&cd);
-	
+	{
+		CORBA_Environment ev;
+		CORBA_exception_init(&ev);
+		goad_server_unregister(NULL, "GOAD:gtcd:19990918",
+			       	       "object", &ev);
+		CORBA_exception_free(&ev);
+	}
 	gtk_main_quit();
 }
 
--- gnome-media-1.2.0/configure.in.gtcdcorba	Sun May 21 16:17:41 2000
+++ gnome-media-1.2.0/configure.in	Thu Jun 15 17:53:01 2000
@@ -9,6 +9,15 @@
 GNOME_INIT
 GNOME_COMPILE_WARNINGS
 
+GNOME_DATADIR=`gnome-config --datadir`
+AC_SUBST(GNOME_DATADIR)
+
+GTCD_GNOME_LIBS=`gnome-config --libs gnomeui gnorba`
+AC_SUBST(GTCD_GNOME_LIBS)
+
+GTCD_GNOME_CFLAGS=`gnome-config --cflags gnomeui gnorba`
+AC_SUBST(GTCD_GNOME_CFLAGS)
+
 AC_PROG_CC
 AC_ISC_POSIX
 
@@ -130,6 +139,10 @@
 
 AC_SUBST(USE_CD_CHANGER)
 AC_SUBST(PROGRAMS_TCD)
+
+dnl CORBA
+
+
 
 dnl
 dnl grecord
