diff -urN aa/gnome-media-2.5.3/configure.in gnome-media-2.5.3/configure.in
--- aa/gnome-media-2.5.3/configure.in	2004-02-24 05:44:59.000000000 +0100
+++ gnome-media-2.5.3/configure.in	2004-02-24 13:55:48.135528432 +0100
@@ -132,7 +132,7 @@
 AM_CONDITIONAL(HAVE_JW, test "x$HAVE_JW" = "xyes")
 AC_SUBST(HAVE_JW)
 
-ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id it ja ko lt lv mk ml mn ms nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tk tr uk vi wa zh_TW zh_CN"
+ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id it ja ko lt lv mk ml mn ms nb nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tk tr uk vi wa zh_TW zh_CN"
 
 AM_GLIB_GNU_GETTEXT
 
