diff -urN gnome-media-2.1.1.orig/cddb-slave2/Makefile.am gnome-media-2.1.1/cddb-slave2/Makefile.am
--- gnome-media-2.1.1.orig/cddb-slave2/Makefile.am	Tue Nov  5 17:18:23 2002
+++ gnome-media-2.1.1/cddb-slave2/Makefile.am	Tue Nov  5 17:19:07 2002
@@ -108,6 +108,5 @@
 EXTRA_DIST = $(server_in_files) $(Desktop_in_files) $(IDLS) $(schema_DATA)
 
 install-schemas:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 
 install-data-local: install-schemas
diff -urN gnome-media-2.1.1.orig/gmix/Makefile.am gnome-media-2.1.1/gmix/Makefile.am
--- gnome-media-2.1.1.orig/gmix/Makefile.am	Tue Nov  5 17:18:23 2002
+++ gnome-media-2.1.1/gmix/Makefile.am	Tue Nov  5 17:19:31 2002
@@ -38,7 +38,6 @@
 schema_DATA=gnome-volume-control.schemas
 
 install-schemas:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 
 install-data-local: install-schemas
 endif
diff -urN gnome-media-2.1.1.orig/gnome-cd/Makefile.am gnome-media-2.1.1/gnome-cd/Makefile.am
--- gnome-media-2.1.1.orig/gnome-cd/Makefile.am	Tue Nov  5 17:18:23 2002
+++ gnome-media-2.1.1/gnome-cd/Makefile.am	Tue Nov  5 17:19:41 2002
@@ -77,6 +77,5 @@
 schema_DATA=gnome-cd.schemas
 
 install-schemas:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(top_builddir)/gnome-cd/gnome-cd.schemas 
 
 install-data-local: install-schemas
diff -urN gnome-media-2.1.1.orig/grecord/Makefile.am gnome-media-2.1.1/grecord/Makefile.am
--- gnome-media-2.1.1.orig/grecord/Makefile.am	Tue Nov  5 17:18:23 2002
+++ gnome-media-2.1.1/grecord/Makefile.am	Tue Nov  5 17:19:50 2002
@@ -32,7 +32,6 @@
 schema_DATA=gnome-sound-recorder.schemas
 
 install-schemas:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(schema_DATA)
 
 install-data-local: install-schemas
 
